component/storage/mysql/store.go

```diff
@@ -416,6 +492,7 @@ func (s *store) Close() error {
 	return nil
 }
 
+// TODO optimize tagMap: https://github.com/hyperledger/aries-framework-go-ext/issues/109
```

Comment:
Note this issue here: https://github.com/hyperledger/aries-framework-go-ext/issues/88

(My intention is to eventually use proper columns+indexing instead of the "tag map")

Reply:
It seems that I forgot to include a reference to #88 in the code... oops...

Reply:
Ok - I closed #109